packages/webview_flutter/webview_flutter_platform_interface/lib/src/types/process_termination_details.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+/// This class provides more specific information about why the render process exited.
+/// The application may use this to decide how to handle the situation.
+@immutable
+class ProcessTerminationDetails {
+  /// Creates a [ProcessTerminationDetails].
+  const ProcessTerminationDetails({
+    required this.didCrash,
+    required this.rendererPriorityAtExit,
+  });
+
+  /// Indicates whether the render process was observed to crash, or whether it was killed by the system.
+  final bool didCrash;
+
+  /// Returns the renderer priority that was set at the time that the renderer exited.
```

Comment:
This is extremely Android-specific, so should not be in this package. If it's actually needed it should be in an Android-specific subclass, but it's not clear to me that there's a plausible use case for this data.